src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -60,12 +62,10 @@ def validate_location(cmd, location):
         if resourceTypes.resource_type == 'connectedClusters':
             rp_locations = [location.replace(" ", "").lower() for location in resourceTypes.locations]
             if location.lower() not in rp_locations:
-                telemetry.set_user_fault()
                 telemetry.set_exception(exception='Location not supported', fault_type=consts.Invalid_Location_Fault_Type,
                                         summary='Provided location is not supported for creating connected clusters')
-                raise CLIError("Connected cluster resource creation is supported only in the following locations: " +
-                               ', '.join(map(str, rp_locations)) +
-                               ". Use the --location flag to specify one of these locations.")
+                raise ArgumentUsageError("Connected cluster resource creation is supported only in the following locations: " +
```

Comment:
Will this automatically get classified as user fault?

Reply:
Yes